docs/excel/custom-functions-naming.md

```diff
@@ -0,0 +1,38 @@
+---
+ms.date: 02/08/2019
+description: Learn requirements for Excel custom functions' names and avoid common naming pitfalls.
+title: Naming guidelines for custom functions in Excel (preview)
+localization_priority: Normal
+---
+# Naming guidelines
+
+A custom function is identified by an `id` and `name` property in the JSON metadata file. The function `id` is used to uniquely identify custom functions in your JavaScript code. The function `name` is used as the display name that appears to a user in Excel. A function name can differ from the function ID, such as for localization purposes. But in general it should stay the same as the ID if there is no compelling reason for them to differ.
```

Comment:
```suggestion
A custom function is identified by an **id** and **name** property in the JSON metadata file. The function id is used to uniquely identify custom functions in your JavaScript code. The function name is used as the display name that appears to a user in Excel. A function name can differ from the function ID, such as for localization purposes. But in general it should stay the same as the ID if there is no compelling reason for them to differ.
```

Reply:
I know we often use code comment formatting around this, but I just noticed in this case it reads oddly. When you see function `name` it almost reads as though `name` is a JavaScript function, because it appears as code. But it is actually a property in JSON. What do you think of just bold on first case then use function name afterwards? Does this work?

Reply:
I agree with David. The property should be bold not code comment formatting. That's actually an MSTP thing!

Reply:
Except I'd continue to bold the property each time it is used.